### ISSUE TITLE
Please provide a proper example

### ISSUE THREAD

**Issue by sandrotosi** [ericmjl/hiveplot#4]
It would be nice to have a proper example you can run and have the output of an hive plot instead of "take your graph, apply this generic rules and then get an hiveplot" as now :)

**Comment by ericmjl** [comment]
Definitely! By the way, do you happen to have a NetworkX graph that can be included as one of the examples? I have my own dummy graph in the test folder, but I would like to include real graphs that are not my own research use cases.

**Comment by sandrotosi** [comment]
ah no, no graph: I was just trying this module and wanted to see what was capable of :) I later noticed the ipython notebook in the test directory, but it looks like a minimal graph not showing much from a user pov (also you might want to remove the build directory from git, as it is an artifact of the module build process which is not part of the source code)

**Comment by ericmjl** [comment]
Gotcha, thanks for the input! I'll include a link to the IPython notebook in the README file as well. 

I think I might be able to improve the description of the use case in the README as well, so that it's a bit clearer.

As for a real graph, still looking around - I'll be able to get one up post PyCon 2015 (after my tutorial session), so that I can leverage the networks created by students in the class.

**Comment by brainstorm** [comment]
If you are interested in creating a "realistic" cancer genomics plot, feel free to use my data and notebook:

https://github.com/brainstorm/cancer-hiveplots/blob/master/hiveplots.ipynb

I did not know this nifty matplotlib integration existed, neat! Are you still mantaining it? Will it be upstream in matplotlib/seaborn sometime in the future?

**Comment by ericmjl** [comment]
@brainstorm I am definitely still maintaining this!

I have wondered about matplotlib integration - I've contributed just a little bit to the matplotlib repo (fixing examples in the documentation, nothing fancy), and asked about this. 

(Caveat: I'm not a very experienced developer, so take my following words with a grain of salt please!)

I think the API specification might take a bit of time and discussion with others. Network visualizations are inherently higher dimensional than other 2-D plots, so it'll take some time.

However, I'm happy to keep hiveplot up separately until network visualizations like Hive/Circos become more mainstream. In the mid-term, I think it makes most sense to build upon matplotlib (and other vis backends, like bokeh).

**Comment by ericmjl** [comment]
@brainstorm Also, thanks for pointing me to your data! I plan to spend some of my off-time over the next few weeks incorporating it in. :smile: Due credit will be given!

**Comment by ericmjl** [comment]
Update for those involved in this discussion, I have uploaded a protein graph example that comes from my own research. Looking forward to putting up an example from @brainstorm 's repository!

**Comment by lucventurini** [comment]
I am having an issue using the notebook:

---

ImportError                               Traceback (most recent call last)
<ipython-input-3-23a10e5067cc> in <module>()
----> 1 p = nx.read_gpickle("Downloads/hiv1_homology_model.pkl")

<decorator-gen-297> in read_gpickle(path)

/usr/users/ga002/venturil/py351/lib/python3.5/site-packages/networkx-1.11-py3.5.egg/networkx/utils/decorators.py in _open_file(func, _args, *_kwargs)
    218         # Finally, we call the original function, making sure to close the fobj.
    219         try:
--> 220             result = func(_new_args, *_kwargs)
    221         finally:
    222             if close_fobj:

/usr/users/ga002/venturil/py351/lib/python3.5/site-packages/networkx-1.11-py3.5.egg/networkx/readwrite/gpickle.py in read_gpickle(path)
     99     .. [1] http://docs.python.org/library/pickle.html
    100     """
--> 101     return pickle.load(path)
    102 
    103 # fixture for nose tests

ImportError: No module named 'pin.pin'

​
I have installed the pin module, but indeed it does not contain any ".pin" submodule.

**Comment by ericmjl** [comment]
@lucventurini: I think you may have installed the `pin` package from PyPI directly. My bad for not providing clearer instructions; it should be [this package](https://github.com/ericmjl/protein-interaction-network), which hasn't been released to PyPI yet.

**Comment by lucventurini** [comment]
Ah cool, thanks. And sorry for the late reply, I actually saw it last week but had not had the time to experiment!
